### ISSUE TITLE
Finetune with triplet loss on small dataset

### ISSUE THREAD

**Issue by WIll-Xu35** [deepinsight/insightface#627]
Hi all,

I'm trying to finetune the pretrained mobilefacenet model with triplet loss on my private dataset, which is quite small (250 images from 25 identities).

But when I finetune it, it seems that the per_batch_size and triplet_bag_size is causing the training to fail, no matter how small I set those two parameters. I'm wondering what does the triplet_bag_size parameter stand for and is it possible to do finetune on such small datasets, if not, what is the smallest dataset size that is acceptable for finetuning?

Many thanks!
